index.html

```diff
@@ -2,9 +2,29 @@
 <html>
   <head>
     <meta charset="utf-8">
-    <title>Hello World</title>
+    <title>Code Coop</title>
+    <link rel="stylesheet" href="assets/css/index.css">
   </head>
   <body>
-      <h1>Hello World!</h1>
+    <header>
+        <h1>Code Coop</h1>
+    </header>
+      <nav>
+        <a href="about.html" title="About page and what we do">About</a>
+        <a href="contact.html" title="Contact page with contact details">Contact Us</a>
```

Comment:
Looks like that the links for "About" and "Contact Us" are swapped around depending on which page you're on.
```html
<!-- index.html -->
<a href="about.html" title="About page and what we do">About</a>
<a href="contact.html" title="Contact page with contact details">Contact Us</a>
```
```html
<!-- contact.html -->
<a href="index.html" title="Home page">About</a>
<a href="about.html" title="About page and what we do">Contact Us</a>
```
```html
<!-- about.html -->
<a href="index.html" title="Home page">About</a>
<a href="contact.html" title="Contact page with contact details">Contact Us</a>
```

Reply:
Sorted out the order/wording of the links.